internal/clientconn/listener.go

```diff
@@ -128,33 +132,25 @@ func Listen(opts *NewListenerOpts) (*Listener, error) {
 	return l, nil
 }
 
+// Listening returns true if the listener is currently listening and accepting new connection.
+//
+// It returns false when listener is stopped
+// or when it is still running with established connections.
```

Comment:
`// or when it is still running with established connections.` It means the listener hasn't started yet right?

Reply:
The opposite: the listener is closed, accept loop exited, but already accepted connection are still active